slack_bot/requirements.txt

```
@@ -1,4 +1,4 @@
 requests
 boto3
 slack_sdk
-langchain==0.0.126
+langchain==0.0.215
```

Let's upgrade to the latest Langchain as of today. 
```suggestion
langchain==0.0.220
```

Since, you have your PR ready, can you also add this change to the `service` dir.